test/rules/tags-check.spec.js

```diff
@@ -1,70 +1,63 @@
-/**
- * Copyright (c) 2015, Yanis Wang <yanis.wang@gmail.com>
- * MIT Licensed
- */
 
-var expect  = require("expect.js");
+const expect  = require("expect.js");
```

Comment:
in this file singlequotes seems to be used
we could format the whole codebase with prettier later, so this wont be an issue for now